tasks/serve/index.ts

```diff
@@ -79,15 +80,20 @@ export class clitask {
         //we need to force pull the docker image, in case the tag was local already (but old content)
         let p = tl.tool('docker').arg('pull').arg(recipe).exec()        
         p.then(()=>{
+            
+            //Get the Docker Options    
+            dockerOptions = tl.getInput('dockerOptions') 
```

Comment:
have you validated if this version of getInput is for optional arguments? Or do you need to use another overload where you tell it that the argument is optional? (don't remember if the default is required or optional).